lib/pure/algorithm.nim

```diff
@@ -556,6 +558,76 @@ proc isSorted*[T](a: openArray[T], order = SortOrder.Ascending): bool =
     assert isSorted(e) == false
   isSorted(a, system.cmp[T], order)
 
+proc merge*[T](
+  x, y: openArray[T], cmp: proc(x, y: T
+): int {.closure.}, order = SortOrder.Ascending): seq[T] {.since: (1, 5, 1).} =
+  ## Merges two sorted `openArray`. All of inputs are assumed to be sorted.
+  ## If you do not wish to provide your own ``cmp``,
```

Comment:
nit: prefer single backticks

Reply:
Yeah, I copy this from the comments of `sort`. I will change it.